apps/internal/json/json.go

```diff
@@ -137,7 +141,6 @@ func callMarshalJSON(v reflect.Value) ([]byte, error) {
 		if v.CanAddr() {
 			v = v.Addr()
 		}
-		panic(fmt.Sprintf("callMarshalJSON called on type %T that does not have MarshalJSON defined", v.Interface()))
```

Comment:
originally a bug or a change in thinking on this?

Reply:
This was a bug.  The if condition didn't even matter there.